packages/web-frontend/src/styles/index.js

```diff
@@ -385,23 +385,6 @@ export const VIEW_STYLES = {
     lineHeight: '16px',
     textAlign: 'left',
   },
-  data: isMobile()
```

Comment:
I checked and this is only used in our component so it's safe to move the styles into our file and delete this shared const. The advantage of having it locally defined is that we can change the styles without having to worrying about if it affects something else and it's just less cognitive load than switching between files.